Wrong postgres username when running `docker-compose` up

When running `docker-compose up` the web image does not start with the below exception.

```
web_1          | Caused by: org.flywaydb.core.api.FlywayException: Unable to obtain Jdbc connection from DataSource
web_1          | 	at org.flywaydb.core.internal.util.jdbc.JdbcUtils.openConnection(JdbcUtils.java:56)
web_1          | 	at org.flywaydb.core.Flyway.execute(Flyway.java:1385)
web_1          | 	at org.flywaydb.core.Flyway.migrate(Flyway.java:1006)
web_1          | 	at org.springframework.boot.autoconfigure.flyway.FlywayMigrationInitializer.afterPropertiesSet(FlywayMigrationInitializer.java:66)
web_1          | 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1637)
web_1          | 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1574)
web_1          | 	... 24 more
web_1          | Caused by: org.postgresql.util.PSQLException: FATAL: role "sa" does not exist
web_1          | 	at org.postgresql.core.v3.ConnectionFactoryImpl.readStartupMessages(ConnectionFactoryImpl.java:691)
web_1          | 	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:207)
web_1          | 	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:65)

```

The problem is caused by missing `""` around json env var. Adding that shows other errors:

```
Caused by: java.lang.IllegalArgumentException: 'credentialsDirectoryPath' does not exist
``` 
even after adding `tokens` config like this:
```
'SPRING_APPLICATION_JSON="{"tokens": {"autoStartup": false, "credentials-directory": "$${user.home}/.berry/pazuzu-registry"}, "spring":{"profiles": {"active": "dev"}, "datasource":{"user": "postgres", "url": "jdbc:postgresql://postgres-db:5432/postgres"}}}"'
```

@Markinhos I don't know much about Spring applications so if you had an idea what I'm missing, i'd appreciate it.

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -7,7 +7,7 @@ services:
     links:
     - postgres-db
     environment:
-    - 'SPRING_APPLICATION_JSON={"spring":{"profiles": {"active": "dev"}, "datasource":{"user": "postgres", "url": "jdbc:postgresql://postgres-db:5432/postgres"}}}'
+    - 'SPRING_APPLICATION_JSON={"spring":{"profiles": {"active": "dev"}, "datasource":{"username": "postgres", "url": "jdbc:postgresql://postgres-db:5432/postgres"}}}'
   postgres-db:
     image: postgres:9.4
     expose:

